src/main/resources/application-dev.yml

```diff
@@ -1,3 +1,6 @@
+server:
+  shutdown: immediate
```

Comment:
It seems we still have the old graceful shutdown stuff in the gradle plugin as well as the new spring boot graceful, so catering for both for now.

Reply:
seems a bit naff - would be better to do
```
configurations {
  implementation { exclude(module = "spring-boot-graceful-shutdown") }
}
```